inc/3rd-party/plugins/autoptimize.php

```diff
@@ -103,9 +122,35 @@ function rocket_activate_autoptimize() {
 	if ( 'on' === get_option( 'autoptimize_css_defer' ) ) {
 		update_rocket_option( 'async_css', 0 );
 	}
+
+	$lazyload = get_option( 'autoptimize_imgopt_settings' );
+
+	if ( 1 === (int) $lazyload['autoptimize_imgopt_checkbox_field_3'] ) {
```

Comment:
For the same reason:
```suggestion
	if ( ! empty( $lazyload['autoptimize_imgopt_checkbox_field_3'] ) ) {
```